pkg/get/tools.go

```diff
@@ -348,28 +348,27 @@ https://storage.googleapis.com/kubernetes-release/release/{{.Version}}/bin/{{$os
 			Owner:       "bitnami-labs",
 			Repo:        "sealed-secrets",
 			Name:        "kubeseal",
-			Version:     "v0.14.1",
+			Version:     "v0.17.2",
```

Comment:
We should remove this version so it can be auto-detected. Can you make that change and test again please?